centaur/src/main/resources/reference.conf

```diff
@@ -56,9 +56,9 @@ centaur {
     genomics.endpoint-url = ${?CROMWELL_BUILD_PAPI_ENDPOINT_URL}
     genomics.location = "us-central1"
     batch.location = "us-central1"
-    auth = "Error: BA-6546 The environment variable CROMWELL_BUILD_PAPI_AUTH_MODE must be set/export pointing to a valid auth such as 'application-default'"
+    auth = "service-account"
```

Comment:
Simplifying the process to run Centaur locally.

I appreciate the principle of these sentinel values that explode when you don't set an environment variable.

But for both `CROMWELL_BUILD_PAPI_AUTH_MODE` and `CROMWELL_BUILD_RESOURCES_DIRECTORY` there's only one value we ever set in whole repo.